index.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+		integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+		crossorigin="anonymous" referrerpolicy="no-referrer" />
+	<link rel="stylesheet" href="/app/static/css/index.css">
+	<script src="/app/static/js/script.js" defer></script>
+	<title>DevOps Meetup</title>
+</head>
+
+<body>
+	<header>
+		<nav>
+			<div class="upper-nav">
+				<ul class="upper">
+					<li><a href="#"><i class="fa-brands fa-facebook-f"></i></a></li>
+					<li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+					<li><a href="#" class="menu-link up">English</a></li>
+					<li><a href="#" class="menu-link up">My Page</a></li>
+					<li><a href="#" class="menu-link up">Logout</a></li>
+				</ul>
+			</div>
+			<div class="lower-nav">
+				<div class="logo">
+					<a href="#"><img src="/app/static/images/ignite.svg" alt="" height="50" width="50"></a>
+					<div class="text">
+						<h3>Ujuzi.io&trade;</h3>
+						<h3 id="nrb-summit">DevOps MeetUp</h3>
+					</div>
+				</div>
+				<div class="lower-menu">
+					<ul class="lower-nav-links">
+						<li><a href="#" class="menu-link lower first">About</a></li>
```

Comment:
- You did a great job, your home page looks like the design provide but kindly make sure you implement the about page also. Here below I list some points please make sure you implement them in your project :grinning:  :grinning:  :grinning: 
-[ ] Kindly make sure your project contains a `Home` page and `About` page.
-[ ] It's possible to navigate between the pages, there's a link from the `Home` page to the `About` page and the opposite too. it means on your home page you will have the link to your about page and on the about page, you will have the link to your home page 
-[ ] Both pages contain the same navbar and footer.

Check the  [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) to see the style and the container of your about page and I provide below some picture :+1: :grinning: :grinning: :grinning: 



![image](https://user-images.githubusercontent.com/95297251/191845378-6a36be31-5050-402f-97f8-f388cb86fb28.png)

![image](https://user-images.githubusercontent.com/95297251/191845021-83f0ff37-d2f8-4800-8346-e2a1ebd8a66e.png)